libs/blocks/faas/faas.js

```diff
@@ -12,6 +61,23 @@ const loadFaas = async (a) => {
   if (faas && faas.closest('.dialog-modal')) {
     faas.querySelector('.faas').classList.add('column2');
   }
+
+  // Accessibility logic
+  const formObserver = new MutationObserver(() => {
+    const faasForm = faas.querySelector('.faas-form');
+    if (!faasForm) return;
+
+    createLiveRegion(faasForm);
+    const validationObserver = createValidationObserver();
+
+    [...faasForm.querySelectorAll('.row')].forEach((row) => {
+      validationObserver.observe(row, { attributes: true, attributeFilter: ['class'] });
+    });
+
+    formObserver.disconnect();
+  });
+
+  formObserver.observe(faas, { childList: true, subtree: true });
```

Comment:
This is a clever approach. Nice work! TBH, it sort of melts my brain.

Reply:
Interesting. Ok. If it's working as expected. I just wanted to call out and record what I saw.